src/web/components/Navigation/NoParticipantAccessView.tsx

```diff
@@ -1,17 +1,52 @@
+import { useLocation, useNavigate } from 'react-router-dom';
+
+import { UserWithParticipantRoles } from '../../../api/services/usersService';
+import { getPathWithParticipant } from '../../utils/urlHelpers';
 import { ParticipantSwitcher } from './ParticipantSwitcher';
 
 import './NoParticipantAccessView.scss';
 
-export function NoParticipantAccessView() {
+type NoParticipantAccessViewProps = Readonly<{
+  user: UserWithParticipantRoles | null;
+}>;
+
+export function NoParticipantAccessView({ user }: NoParticipantAccessViewProps) {
+  const navigate = useNavigate();
+  const location = useLocation();
+
+  const onBackToParticipant = () => {
+    let participantId;
+    if (user?.participants) {
+      participantId = user.participants[0].id;
+    } else {
+      participantId = localStorage.getItem('lastSelectedParticipantId');
```

Comment:
shouldn't it be the first option to get the participant from localstorage?

Reply:
Switched the option order!